website/templates/project/modal_remove_contributor.mako

```diff
@@ -0,0 +1,130 @@
+<link rel="stylesheet" href='/static/css/pages/remove-contributor-page.css'>
+<div id="removeContributor" class="modal fade">
+    <div data-bind="css: modalSize">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h3 class="modal-title" data-bind="text:pageTitle"></h3>
+            </div>
+            <div class="modal-body" >
+
+                <div data-bind="if: canRemoveNode() && !pageChanged()">
+                    <!-- remove page -->
+                    <div data-bind='if:page() === REMOVE'>
+                        <div class="form-group">
+                            <span>Do you want to remove <b data-bind="text:removeSelf() ? 'yourself' : contributorToRemove()['fullname']"></b> from
+                                <b>{{title}}</b>, or from <b>{{title}}</b> and every component in it?</span>
+                        </div>
+                        <div id="remove-page-radio-buttons" class="col-md-8" align="left">
+                            <div class="radio">
+                                <label><input type="radio" name="radioBoxGroup" data-bind="checked:deleteAll, checkedValue: false" checked>
+                                    Remove <b data-bind="text:removeSelf() ? 'yourself' : contributorToRemove()['fullname']"></b> from
+                                    <span class="f-w-lg" data-bind="text: title"></span>
```

Comment:
Needs a period at the end of the sentence.

![screen shot 2016-01-15 at 12 14 46 pm](https://cloud.githubusercontent.com/assets/134455/12359644/959cd0f6-bb81-11e5-9e3e-3365910d3bcc.png)